Assets/Scripts/Utility/MacroHelper.cs

```diff
@@ -228,12 +229,22 @@ public static class MacroHelper
             { "%wth", Worth }, // Worth
         // DF Unity - new macros:
             { "%pg", PlayerPronoun },   // He/She (player)
-            { "%pg1", PlayerPronoun },  // His/Her (player)
+            { "%pg1", PlayerPronoun },  // He/She (player)
```

Comment:
This line should be changed to:

`{ "%pg1", PlayerPronoun1 },  // He/She (player)`

and the corresponding method not removed further down. Careless typo by me from Sept 2017.

Alternatively just leave both alone and I'll submit a tiny fix PR.